resources/remote_management.rb

```diff
@@ -0,0 +1,24 @@
+resource_name :remote_management
+default_action :enable
+
+kickstart = '/System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/Resources/kickstart'
+
+action :enable do
+  execute "#{kickstart} -activate" do
```

Comment:
Any particular reason we chose to use guards here instead of `load_current_value`?

Reply:
There aren't any properties to `load_current_value` of.

Reply:
Are actions actually just properties?

Reply:
Is something like this possible?
```
load_current_value do
    enable RemoteManagement.activated?
end

action :enable do
  converge_if_changed :enable do
    execute "#{kickstart} -activate"
  end
end